### PR TITLE
Feature/orbs n points

### PR DESCRIPTION
**Pull request by lucastswick**
- Fixes some layout issues with OrbManager
- Orbs dispatch events when 'complete'
- Points are incremented when user 'completes' orbs
- User reducer.

### CODE REVIEW

File: packages/project-disaster-trail/src/components/Game/Orb.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /** @jsx jsx */
-import { Component } from "react";
+import { PureComponent } from "react";
 import { jsx, css } from "@emotion/core";
+import { returnStatement } from "@babel/types";
```
Comment (jaronheard) [inline packages/project-disaster-trail/src/components/Game/Orb.js]: What's this do?

Comment (lucastswick) [reply packages/project-disaster-trail/src/components/Game/Orb.js]: I will remove that. Sometimes my auto-imports give me whack results.

File: packages/project-disaster-trail/src/components/Game/Orb.js [hunk 2]
```diff
@@ -92,41 +88,44 @@ class Orb extends Component {
...
     `;
 
-    const orbSizeStyle = css`
-      height: ${orbSize + 20}px;
-      width: ${orbSize + 20}px;
-      border-radius: ${orbSize + 20}px;
+    const absoluteStyle = css`
+      position: absolute;
+      top: -${size / 2}px;
+      left: -${size / 2}px;
```
Comment (Dianna) [inline packages/project-disaster-trail/src/components/Game/Orb.js]: Nice. I was so confused why things were in weird places

File: packages/project-disaster-trail/src/components/Game/OrbManager.js [hunk 3]
```diff
@@ -4,29 +4,44 @@ import { map } from "lodash";
...
-
-const OrbManager = ({ ratios }) => {
+/**
+ * OrbManager is responsible for moving Orbs
+ *
+ * @param {*} { orbCount=2 } how many orbs should be rendered?
+ * @param {*} { velocityX=2 } how fast does the orb move horizontally
+ * @param {*} { velocityY=0 } how fast does the orb move vertically
+ * @param {*} { period=2 } how much vertical modulation per frame should the orb move? Higher number = more 'wobbly'
+ * @returns
```
Comment (Dianna) [inline packages/project-disaster-trail/src/components/Game/OrbManager.js]: Great documentation

File: packages/project-disaster-trail/src/components/Game/OrbManager.js [hunk 4]
```diff
@@ -4,29 +4,44 @@ import { map } from "lodash";
...
   let tempModels = [];
 
+  const { interfaceHeight } = ratios;
+
   // tick is used to modulate movement based on an incrementing value
   let tick = 0;
 
-  // how big is the circle the orbs 'float' up and down in
-  const period = 2;
+  // const period = 2;
```
Comment (Dianna) [inline packages/project-disaster-trail/src/components/Game/OrbManager.js]: Can this be removed?

File: packages/project-disaster-trail/src/components/Game/RadialGauge.js [hunk 5]
```diff
@@ -47,13 +34,27 @@ class RadialGauge extends Component {
...
+    const { animateGauge, size } = this.props;
     const { percent } = this.state;
 
+    const gaugeDefaultStyle = css`
+      transition: transform 1s;
+      position: absolute;
+      top: ${size / 2}px;
+      left: ${size / 2}px;
+      height: ${size}px;
+      width: ${size}px;
```
Comment (Dianna) [inline packages/project-disaster-trail/src/components/Game/RadialGauge.js]: Thanks for getting that sizing hooked in.

Comment (lucastswick) [reply packages/project-disaster-trail/src/components/Game/RadialGauge.js]: You betcha!

File: packages/project-disaster-trail/src/components/Game/OrbManager.js (hunk 4 above)
Comment (lucastswick) [reply packages/project-disaster-trail/src/components/Game/OrbManager.js]: Eagle eye!

File: packages/project-disaster-trail/src/components/Game/OrbManager.js (hunk 3 above)
Comment (lucastswick) [reply packages/project-disaster-trail/src/components/Game/OrbManager.js]: Thanks!

File: packages/project-disaster-trail/src/components/Game/Orb.js (hunk 2 above)
Comment (lucastswick) [reply packages/project-disaster-trail/src/components/Game/Orb.js]: Silly css layouts are always a pain!

File: packages/project-disaster-trail/src/components/Game/Orb.js (hunk 1 above)
Comment (lucastswick) [reply packages/project-disaster-trail/src/components/Game/Orb.js]: Removed!

**Pull request merged**